Artifacts/linux-deprovision/depro.sh

```diff
@@ -11,7 +11,6 @@ main () {
             apt-get -y update > /dev/null
             apt-get -y install at > /dev/null
         elif [ -n "$isYum" ] ; then
-            yum -y update > /dev/null
```

Comment:
What's the outcome of not running this line?

Reply:
It should all work as expected, I'm testing this script on every family of linux we have to make sure it doesn't break anything.